### PR TITLE
DOC: clean v1.1.1 release notes

### PR DESCRIPTION
**Pull request by simonjayhawkins**

### CODE REVIEW

**Comment by simonjayhawkins** [comment]
@TomAugspurger I don't think we need the subheaders for bugfixes in patch releases

**Comment by TomAugspurger** [comment]
Looks good.

**Review by WillAyd — APPROVED** [review]

File: doc/source/whatsnew/v1.1.1.rst [hunk 1]
```diff
@@ -15,20 +15,23 @@ including other versions of pandas.
...
 - Fixed regression in :meth:`DataFrame.apply` where functions that altered the input in-place only operated on a single row (:issue:`35462`)
 - Fixed regression in :meth:`DataFrame.reset_index` would raise a ``ValueError`` on empty :class:`DataFrame` with a :class:`MultiIndex` with a ``datetime64`` dtype level (:issue:`35606`, :issue:`35657`)
-- Fixed regression where :meth:`DataFrame.merge_asof` would raise a ``UnboundLocalError`` when ``left_index`` , ``right_index`` and ``tolerance`` were set (:issue:`35558`)
+- Fixed regression where :func:`pandas.merge_asof` would raise a ``UnboundLocalError`` when ``left_index`` , ``right_index`` and ``tolerance`` were set (:issue:`35558`)
 - Fixed regression in ``.groupby(..).rolling(..)`` where a custom ``BaseIndexer`` would be ignored (:issue:`35557`)
 - Fixed regression in :meth:`DataFrame.replace` and :meth:`Series.replace` where compiled regular expressions would be ignored during replacement (:issue:`35680`)
 - Fixed regression in :meth:`~pandas.core.groupby.DataFrameGroupBy.agg` where a list of functions would produce the wrong results if at least one of the functions did not aggregate. (:issue:`35490`)
```
Comment (jorisvandenbossche) [inline doc/source/whatsnew/v1.1.1.rst:33]:
```suggestion
- Fixed regression in :meth:`~pandas.core.groupby.DataFrameGroupBy.aggregate` where a list of functions would produce the wrong results if at least one of the functions did not aggregate. (:issue:`35490`)
```

(the agg version doesn't seem to exist in the API docs)

File: doc/source/whatsnew/v1.1.1.rst [hunk 2]
```diff
@@ -37,50 +40,11 @@ Fixed regressions
...
-
-
-**Numeric**
-
--
--
-
-**Groupby/resample/rolling**
-
 - Bug in :class:`pandas.core.groupby.RollingGroupby` where passing ``closed`` with column selection would raise a ``ValueError`` (:issue:`35549`)
```
Comment (jorisvandenbossche) [inline doc/source/whatsnew/v1.1.1.rst:46]: Same comment here like above

**Review by jorisvandenbossche — COMMENTED** [review]
Looks good! Added a few comments on dysfunctional API links, but those can also be done as follow-up (since not strictly related to the changes you did)

**Comment by simonjayhawkins** [comment]
Thanks @jorisvandenbossche I've also updated `Styler` link.

That should just leave and `CategoricalIndex.format()` as dysfunctional (will do final check after merging this before adding release date) 

https://pandas.pydata.org/docs/dev/whatsnew/v1.1.1.html

Could link to Index.format() instead and re-write note for context.

**Review by jorisvandenbossche — APPROVED** [review]

**Pull request merged**